Documentation/security.md

```diff
@@ -143,6 +143,14 @@ $ etcd -name infra2 -data-dir infra2 \
 
 The etcd members will form a cluster and all communication between members in the cluster will be encrypted and authenticated using the client certificates. You will see in the output of etcd that the addresses it connects to use HTTPS.
 
+## Notes For etcd Proxy
+
+etcd proxy terminates the TLS from its client if the connection is secure, and uses proxy's own key/cert specified in `--peer-key-file` and `--peer-cert-file` to communicate with etcd members.
+
+Proxy communicates with etcd member through both advertise client urls and advertise peer urls of that member. It forwards the client requests to etcd members’ advertise client urls, and it syncs the initial cluster configuration through etcd members’ advertise peer urls.
```

Comment:
"The proxy communicates with etcd members through both the `--advertise-client-urls` and `--advertise-peer-urls` of a given member. It forwards client requests to etcd members’ advertised client urls, and it syncs the initial cluster configuration through etcd members’ advertised peer urls."